notification/migrations/0001_initial.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 4.2 on 2023-05-24 12:03
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('player', '0002_static_players'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Notification',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('seen_time', models.DateTimeField(blank=True, null=True)),
+                ('sent_time', models.DateTimeField(default=django.utils.timezone.now)),
+                ('message', models.TextField(blank=True, null=True)),
+                ('notification_type', models.CharField(choices=[('website', 'Website'), ('game-event', 'Game-Event')],
```

Comment:
As I saw the other fields here, I assume that you want to use the notification in the chat (correct me if I wrong).
If so, you don't need here type of 'chat'?

Reply:
Added, thanks for notice that